### DISCUSSION TITLE
Zed incorrectly inserts closing angle bracket for common div creation pattern

### DISCUSSION THREAD

**Discussion by defjosiah** [zed-industries/zed#13813] [Feature Requests]
Starting with this string
```
<div class="test"></div
```
When I close the angle bracket at the end of the `</div`, Zed inserts a character in the starting div
```
<div> class="test"></div>
```
This creates an incorrect div, and is really frustrating to fix.

I do not think that this is related to any of the autoclose settings (I've toggled "use_autoclose", and it happens in both cases).
I am using svelte, so there's a chance this is something to do with an odd LSP server, but it feels like built-in behavior.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
